src/scripts/KeyBinds.ts

```diff
@@ -1,7 +1,7 @@
-$(document).ready(function() {
-    $(document).on("keydown", function(e) {
+$(document).ready(function () {
+    $(document).on("keydown", function (e) {
         let keyCode = e.keyCode;
-        if (Game.gameState() == GameConstants.GameState.safari) {
+        if (App.game.gameState == GameConstants.GameState.safari) {
```

Comment:
Other gameState checks use `===`, may as well make it consistent while we are here. Same "issue" in the keyup handler.